hw/ip_templates/otp_ctrl/doc/programmers_guide.md

```diff
@@ -111,6 +111,31 @@ The hardware will set [`DIRECT_ACCESS_REGWEN`](registers.md#direct_access_regwen
 It should also be noted that the effect of locking a partition via the digest only takes effect **after** the next system reset.
 To prevent integrity check failures SW must therefore ensure that no more programming operations are issued to the affected partition after initiating the digest calculation sequence.
 
+## Zeroization Sequence
+
+Zeroizing a partition is completely handled by firmware and resembles an ordinary in-field provisioning flow.
```

Comment:
Could we tweak the 'completely handled by firmware' section here? From the perspective of the OTP Controller DAI interface, who drives it / how it is driven is really out of scope. Also, I kind of dislike 'firmware' as a catch-all term.
How about this?
```suggestion
A partition zeroization sequence is driven by accesses through the OTP Controller's DAI interface, and resembles an ordinary in-field provisioning flow.
```